### PR TITLE
Inji 637:  data backup restore

### PR DESCRIPTION
**Pull request by Alka1703**

### CODE REVIEW

File: machines/backup.ts [hunk 1]
```diff
@@ -2,10 +2,8 @@ import {EventFrom, StateFrom, send} from 'xstate';
 import {createModel} from 'xstate/lib/model';
 import {AppServices} from '../shared/GlobalContext';
 import {StoreEvents} from './store';
-import Storage, {
-  isMinimumLimitForBackupReached,
-  writeToBackupFile,
-} from '../shared/storage';
+import {isMinimumLimitForBackupReached} from '../shared/storage';
+import {writeToBackupFile} from '../shared/fileStorage';
 import {compressAndRemoveFile} from '../shared/fileStorage';
```
Comment (swatigoel) [inline machines/backup.ts:7]: Combine imports for `../shared/fileStorage'`

**Review by sree96 — COMMENTED** [review]
Plz look into the comments by Swati.

**Review by vharsh — APPROVED** [review]
lgtm

**Review by vharsh — APPROVED** [review]
lgtm

**Pull request merged**